Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-button/ContestButton.module.scss

```diff
@@ -0,0 +1,3 @@
+﻿.contestButton {
```

Comment:
Rename to `contestButtonFaint` or something similar, to indicate it's for reducing opacity.

Reply:
Done.